README.md

```diff
@@ -42,10 +42,39 @@ $ npm test
 ```bash
 $ ./bin/test.sh
 ```
+## Kafka transaction tests
 
-You can run the integration tests with:
-```bash
-$ ./e2e/test.sh
+
+1. Run the needed infrastructure
+
+In one console execute the command:
+
+```
+./e2e/start-environment.sh
+```
+
+2. Run the test itself
+
+In another console execute the test:
+
+```
+./e2e/test.sh
```

Comment:
This test was not working. Resurrect it, but make it run outside of the docker compose. This helps to debug the test with a debugger or to analyze if the infrastructure is not properly initialized.

Reply:
I can see no way to maintain this test if we don't run it in Jenkins. Can we run it every build?

Reply:
Why would you run integration tests with every build? Isn't that reserved for unit tests?
--
On the PR side, I saw the `e2e` folder a while ago and realised quite later that those were some tests. Didn't give it much thought as it didn't work, but now that we're on it - shouldn't all tests be in the `test` folder and that folder should be separated into `unit` and `integration`? Gives more clarity to the structure IMO.

Reply:
I am not sure we need that test to run all the time, it is more of a test of node-rdkafka and demo how to use transactions. I added this code back when I added a [PR](https://github.com/Blizzard/node-rdkafka/pull/881) for transactions support in node-rdkafka.